worlds/sm64ex/Rules.py

```diff
@@ -117,7 +122,7 @@ def set_rules(world, player: int, area_connections):
     add_rule(world.get_location("Toad (Third Floor)", player), lambda state: state.can_reach("Third Floor", 'Region', player) and state.has("Power Star", player, 35))
 
     if world.MIPS1Cost[player].value > world.MIPS2Cost[player].value:
-        world.MIPS2Cost[player].value = world.MIPS1Cost[player].value
+        (world.MIPS2Cost[player].value, world.MIPS1Cost[player].value) = (world.MIPS1Cost[player].value, world.MIPS2Cost[player].value)
```

Comment:
Why?

Reply:
So that you can select `random` (or `random-range`s with overlapping ranges) on them without a disproportionate chance of them ending up as the same value

Reply:
Alright, makes sense